### PR TITLE
Feature navigation

### PR DESCRIPTION
**Pull request by fortuneonyeka**
In this project milestone, we modified awesome book app to have the following :
A Navigation bar.
Three content sections:
Books list.
Add book form.
Contact info.
current date below the navigation bar.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -7,6 +7,33 @@ const title = document.querySelector('#title');
 const author = document.querySelector('#author');
 const btn = document.querySelector('.btn');
 const bookHolder = document.querySelector('#bookContainer');
+const list = document.querySelector('#list');
+const add = document.querySelector('#add');
+const contact = document.querySelector('#contact');
+
+const dt = new Date();
```
Comment (ZawadiSumba66) [inline main.js]: - [x] You are supposed to implement the date logic using the luxon library as from the requirements. [luxon](https://moment.github.io/luxon/docs/manual/install.html)

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
Hi Team 😃 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (rloterh) [reply main.js]: This issue is well resolved! ✔️

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (rloterh) [inline .gitignore:1]: - [ ] OPTIONAL
To enhance a more professional repository kindly ensure to add the `.vscode` folder in the `.gitignore` file.

**Your current project repository file structure:**
image

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Ngalula & Fortune, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 🍾 :shipit: .
Congratulations! 🎉 🎉 🥂

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comment(s) marked **[OPTIONAL]** under this review:_

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**